libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini/defaults.parm

```diff
@@ -1,7 +1,7 @@
 # setup for LEDs on chan9
 SERVO9_FUNCTION 120
 # Default VTX power to on
-RELAY_DEFAULT 1
+RELAY2_DEFAULT 1
```

Comment:
Shouldn't this be RELAY1_DEFAULT ?

Reply:
It uses relay 2. 

https://github.com/ArduPilot/ardupilot/blob/1d945cc5ac8b6555e0c86500653b74fb41106111/libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini/hwdef.dat#L75-L77

It seems to be a pattern for lots of the boards, I guess to leave relay 1 free for user stuff.